first_challenge.exs

```diff
@@ -0,0 +1,42 @@
+{limit1, _} = IO.gets("Enter the first number: ") |> Integer.parse
+{limit2, _} = IO.gets("Enter the second number: ") |> Integer.parse
+
+defmodule Calculatesequence do
+  def sequence(limit,number) when limit > 0 do
+    number = "#{limit}" <> number
+    sequence(limit-1,number)
+  end
+
+  def sequence(_limit,number) do
+    number
+  end
+end
+
+defmodule Math do
+  def isDivisor(n,cont) do
+    cond do
+      rem(n, 3) == 0 -> _cont = cont + 1
+      true -> cont
+    end
+  end
+end
+
+
+defmodule Main do
+  def main(n1,n2,cont,_number) when n1 <= n2 do
```

Comment:
You can also use default parameters for avoiding passing values such as `""` or `0` when invoking this function